packages/lesswrong/server/emails/renderEmail.tsx

```diff
@@ -72,7 +72,7 @@ const emailGlobalCss = `
   
   /* Global styles that apply eg inside of posts */
   a {
-    color: ${isEAForum ? '#0C869B' : '#5f9b65' // TODO: make this an instance setting?}
+    color: ${getForumTheme({name: "default"}).palette.primary.main};
```

Comment:
Have you tested this by sending an email?

Reply:
I'm having trouble persuading my dev setup to send emails— @jpaddison3, might it be easy for you to check this? If not, I can probably get to this tomorrow.

Reply:
I tested it in the email previewer, IIRC. I'll test it now.

... and it's good.